orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -198,21 +198,21 @@
              {% if 'ATLAS' in exposed and 'ATLAS_SERVER' in salt['pillar.get']('gateway:location') -%}
              <param>
                  <name>ATLAS</name>
-                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySessions=true</value>
              </param>
              <param>
                  <name>ATLAS-API</name>
-                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySessions=true</value>
              </param>
              {%- endif %}
              {% if 'CM-API' in exposed and 'CM-API' in salt['pillar.get']('gateway:location') -%}
              <param>
                  <name>CM-API</name>
-                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySessions=true</value>
              </param>
              <param>
                  <name>CM-UI</name>
-                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
+                 <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000;enableStickySessions=true</value>
              </param>
              {%- endif %}
              {% if 'HIVE' in exposed and 'HIVESERVER2' in salt['pillar.get']('gateway:location') -%}
```

Comment:
Why are we not this for HIVE? Hive would be one to benefit the most.

Reply:
Hive is connected to by the compute engines, so it shouldn't need this?  Or should it?  I've only done this for the ui/api services.

Also it's possible that the datahub would use those configurations as well, I'm not sure if they should be supported?

Reply:
The original bug was opened for Hive and I tested this, like dev tested it using beeline for hive.

Reply:
@orngejaket We want to change only for Datalake right? Is the current code change applicable for Datalake or Datahubs as well?

@moresandeep I understand that the original one was opened for Hive, but we'd like to use it for Atlas/Ranger and the other components in the datalake.

Reply:
We should be careful with this, there are some edge cases that could cause issues with retries. 
Also, Atlas is active/passive so HA will not work there.